### PR TITLE
Initial push to Artifactory

### PR DESCRIPTION
**Pull request by payam-delfi**
## Changes

- Initial push to Artifactory: the GH Actions only kicks in if there is a change in the DESCRIPTION file, so I'm upversioning this to make sure everything is working.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Package: clusterPower
-Version: 0.7.1
+Version: 0.7.2
```
Comment (sbanville-delfi) [inline DESCRIPTION:2]: **Question:** Is there a "standard" way to indicate this version is a Delfi version and not a `clusterPower` version ?

Comment (sbanville-delfi) [reply DESCRIPTION:2]: This is not a show stopper, so I am approving.

**Comment by sbanville-delfi** [comment]
LGTM. See my comment.

**Comment by sbanville-delfi** [comment]
Correction, I see it was merged :-)

**Comment by payam-delfi** [comment]
> Correction, I see it was merged :-)

Oh yeah, sorry I ~was~ am trying to get the infra working. The push to Artifactory is giving me some trouble. 😅

**Pull request merged**